src/App/routes.tsx

```diff
@@ -1101,6 +1101,36 @@ const fieldReportFormNew = customWrapRoute({
     },
 });
 
+const fieldReportsFormDetails = customWrapRoute({
+    path: 'field-reports/:fieldReportId',
+    component: {
+        render: () => import('#views/FieldReportDetails'),
+        props: {},
+    },
+    parent: root,
+    wrapperComponent: Auth,
+    context: {
+        title: 'Field Report Details',
+        visibility: 'anything',
+    },
+});
+
+// FIXME: Add field report form
+
+const fieldReportsFormEdit = customWrapRoute({
+    path: 'field-reports/:fieldReportId/edit',
+    component: {
+        render: () => import('#views/DrefApplicationForm'),
```

Comment:
We already have a dummy view for it
```suggestion
        render: () => import('#views/FieldReportForm'),
```